clm_demeter/prepare_baselayer.py

```diff
@@ -0,0 +1,161 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt # to generate plots
+from netCDF4 import Dataset     # to work with NetCDF files
+import os                       # operating system interface
+from scipy.interpolate import griddata
+
+__author__ = 'Min Chen and Eva Sinha'
+__email__ = 'min.chen@pnnl.gov and eva.sinha@pnnl.gov'
+__copyright__ = 'Copyright (c) 2017, Battelle Memorial Institute'
+
+# ------------------------------------------------------------------------
+def interpolate_coarse_fine(data, coarse_grid_coord, fine_grid_cooord):
+    """ Interpolate values of variable from coarse grid to fine grid
+    :param data:             subset(only land cells) of coarse grid coordinates along with values for variable
+    :param coarse_grid_coord pandas dataframe containing latitude and longitude coordinate of coarse grid
+    :param fine_grid_coord   pandas dataframe containing latitude and longitude coordinate of fine grid
+    
```

Comment:
Need to document what the function is returning by inserting `:return:     <your explanation>` after your parameter descriptions.